recipes/seq2c/meta.yaml

```diff
@@ -27,3 +28,6 @@ about:
   home: https://github.com/AstraZeneca-NGS/Seq2C
   license: MIT
```

Comment:
Additionally, since the license file is already in the tarball, you can also add `license_file` and `license_family` to the recipe.